compiler/mio-tf/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 nnas_find_package(Protobuf QUIET)
 # TensorFlowSource package is used to use ~.proto files
-nnas_find_package(TensorFlowSource EXACT 1.12 QUIET)
+nnas_find_package(TensorFlowSource EXACT 2.3 QUIET)
```

Comment:
Currently mio-tf is not active project and not sure changing to 2.3 is valid. (I assume this is to remove 1.12 source related things)
I may think future me or others should do the validation :)